src/network/clientpackethandler.cpp

```diff
@@ -819,6 +819,36 @@ void Client::handleCommand_StopSound(NetworkPacket* pkt)
 	}
 }
 
+
+
+void Client::handleCommand_PauseSound(NetworkPacket* pkt)
+{
+	s32 server_id;
+
+	*pkt >> server_id;
+
+	UNORDERED_MAP<s32, int>::iterator i = m_sounds_server_to_client.find(server_id);
```

Comment:
@rubenwardy this one is the same as every original iterator name in the minetest sound api stuff.
If this is inaccurate, should I change every iterator I've seen labeled as `i` (like in some functions above this one)?